composer.json

```diff
@@ -14,26 +15,35 @@
         "irc": "irc://irc.freenode.net/yii",
         "source": "https://github.com/yiisoft/_____"
     },
+    "minimum-stability": "dev",
     "require": {
-        "php": "^7.4"
+        "php": "^7.4",
+        "psr/event-dispatcher": "1.0.0"
     },
     "require-dev": {
+        "hiqdev/composer-config-plugin": "^1.0@dev",
```

Comment:
May be `hiqdev/composer-config-plugin` should be placed in the `require` section?

Reply:
It's not used in `src`.

Reply:
What about `config/common.php` and `extra->config-plugin` directives in the `composer.json`?

Reply:
These would be ignored except in the case when this package is used in another package having "config-plugin" as dependency/